Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -983,6 +1150,46 @@ def top_tags_results_command():
     })
 
 
+def search_ip_command(ip):
+    indicator_type = 'IP'
+    ip_list = argToList(ip)
+    for ip_address in ip_list:
+        raw_res = search_indicator('ipv4_address', ip_address)
+        score = calculate_dbot_score(raw_res, indicator_type)
+        res = parse_indicator_response(raw_res, indicator_type)
+        get_indicator_outputs(indicator_type, res, ip_address, score, 'Address', raw_res)
+
+
+def search_domain_command(domain):
+    indicator_type = 'Domain'
+    domain_list = argToList(domain)
+    for domain in domain_list:
+        raw_res = search_indicator('domain', domain)
+        score = calculate_dbot_score(raw_res, indicator_type)
+        res = parse_indicator_response(raw_res, indicator_type)
+        get_indicator_outputs(indicator_type, res, domain, score, 'Name', raw_res)
+
+
+def search_url_command(url):
+    indicator_type = 'URL'
+    url_list = argToList(url)
+    for url in url_list:
+        raw_res = search_indicator('url', url)
+        score = calculate_dbot_score(raw_res, indicator_type)
+        res = parse_indicator_response(raw_res, indicator_type)
+        get_indicator_outputs(indicator_type, res, url, score, 'Data', raw_res)
+
+
+def search_file_command(file):
+    indicator_type = 'File'
+    file_list = argToList(file)
+    for file in file_list:
```

Comment:
you are shadowing the function parameter, use `file_` in the loop.
plus, loop should cover only `search_indicator`